values-elasticsearch.yaml

```diff
@@ -0,0 +1,22 @@
+replicas: 1
+minimumMasterNodes: 1
+
+resources:
+  requests:
+    cpu: "100m"
+    memory: "2Gi"
+  limits:
+    cpu: "1000m"
+    memory: "2Gi"
```

Comment:
I think, ES' template otherwise sets the cpu request to 1000m by default, so at least that part, we need to set in order to work locally with the 4CPUs that we had in the example.

As for the memory, I lack experience with ES here, feel free to adapt, I went with the defaults...